webapp/components/CommentList/CommentList.vue

```diff
@@ -36,6 +36,11 @@ export default {
     routeHash: { type: String, default: () => '' },
     post: { type: Object, default: () => {} },
   },
+  computed: {
+    postComments() {
+      return (this.post && this.post.comments) || []
+    },
+  },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-18T21:50:48Z" title="Tuesday, February 18th 2020, 10:50:48 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -36,6 +36,11 @@ export default {
     routeHash: { type: String, default: () => '' },
     post: { type: Object, default: () => {} },
   },
+  computed: {
+    postComments() {
+      return (this.post && this.post.comments) || []
+    },
+  },
```


I like this! because you are returning either the `post.comments` array or an empty array, you don't need the `v-if` in the html template